etcdserver/metrics.go

```diff
@@ -112,6 +112,13 @@ var (
 		Help:      "Which Go version server is running with. 1 for 'server_go_version' label with current version.",
 	},
 		[]string{"server_go_version"})
+	serverID = prometheus.NewGaugeVec(prometheus.GaugeOpts{
+		Namespace: "etcd",
+		Subsystem: "server",
+		Name:      "id",
+		Help:      "Server or member ID in hexadecimal format (1 for 'server_id' label with current ID)",
```

Comment:
@gyuho lgtm but, nit, suggest to keep the doc consistent, i.e. two sentences instead of brackets, similar to hasLeader, isLeader, currentVersion etc.